README.md

```diff
@@ -111,6 +104,23 @@ make run
   - string 
   - default: "redis:6379"
   - hostname and port for the redis service
+- UpdateIntervalSeconds
+  - int64
+  - default: 60
+  - Used only in `stream` mode, the interval between requests to fetch blacklisted IPs from LAPI
+- DefaultDecisionSeconds
+  - int64
+  - default: 60
+  - Used only in `live` mode, decision duration of accepted IPs
+- CrowdsecCapiLogin
+  - string
+  - Used only in `alone` mode, login for Crowdsec CAPI
+- CrowdsecCapiPwd
+  - string
+  - Used only in `alone` mode, password for Crowdsec CAPI
```

Comment:
I think Username:Password would be better as naming for ending variables prefixed by CrowdsecCapi